addon/utils/is-descriptor.js

```diff
@@ -2,7 +2,14 @@ import Ember from 'ember';
 let { typeOf, Descriptor } = Ember;
 
 // credit: https://github.com/cibernox/ember-cpm/blob/master/addon/utils.js#L17-L20
-function isDescriptor(prop) {
+function isDescriptor(propertyName) {
```

Comment:
@stefanpenner do you know of a better way to handle this from 1.10->1.13?  It's to determine if a property is a computed property or not.